README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
Ensure that each shell command is properly formatted in a code block for better readability and syntax highlighting. You may want to use triple backticks for both the shell code blocks.